bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+define p = g u c d s u~ c~ d~ s~ b b~ 
+define j = g u c d s u~ c~ d~ s~ b b~ 
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+
+import model sm
```

Comment:
Please remove this unused line import model sm
and if you define p j l+ l- or whatsoever, and then do import model ALP,
the definitions will be rewritten so you should put import model ALP at the top

Reply:
Fix it, thank you!